.storybook/main.js

```diff
@@ -1,15 +1,21 @@
-// Note: addon-measure is forced to false in order to avoid it enabled in ZeroHeight
+/** @type { import('@storybook/html-vite').StorybookConfig } */
 const config = {
-  "stories": ["../stories/**/*.stories.@(js|jsx|ts|tsx)"],
-  "addons": ["@storybook/addon-a11y", {
-    name: "@storybook/addon-essentials",
-    options: {
```

Comment:
Double-check here if https://storybook.js.org/docs/essentials/measure-and-outline is different now